.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "singleQuote": true,
   "printWidth": 120,
-  "arrowParens": "avoid"
+  "arrowParens": "avoid",
+  "trailingComma": "es5"
```

Comment:
Temporary apply Prettier v2 behavior. This adds changes to 700+ files, so I am planning to do it gradually per directory later